imports/ui/signup/signup.actions.test.js

```diff
@@ -45,26 +40,25 @@ if (Meteor.isClient) {
           expect(Accounts.createUser).to.have.been.calledWithMatch({
             email: info.emailAddress,
             password: info.password,
+            tac: {
+              acceptedAt: new Date(),
+              version: '08/06/2018'
+            },
             profile: {
-              phone: info.phoneNumber,
-              name: info.fullName,
-              country: info.country,
-              bzLogin: info.bzLogin,
-              bzPass: info.bzPass
             }
           })
         })
         // Needs implementation and testing
-        it.skip('should dispatch "error" if the API yields an error', () => {})
-        it('should dispatch a navigation event to "/case" if the API call is successful', () => {
+        it('should dispatch "error" if the API yields an error', () => {})
```

Comment:
I'd prefer if you kept it with "skip". It highlights that the test wasn't implemented yet